objects/Cargo.toml

```diff
@@ -31,6 +31,7 @@ log = { version = "0.4", optional = true }
 miden-crypto = { workspace = true }
 miden-verifier = { workspace = true }
 serde = { version = "1.0", optional = true, default-features = false, features = ["derive"] }
+thiserror = { version = "1.0" }
```

Comment:
Are we using `thiserror` anywhere? If not, let's remove it from here.

Reply:
removed